src/app/project/project.component.html

```diff
@@ -89,27 +89,52 @@ <h3>{{project.commentPeriodForBanner.dateStarted | date: 'MMMM d'}} -
 
     <section class="people">
       <div class="container">
-        <div class="flex-container">
-          <div class="inner-col">
-            <div *ngIf="project?.contactFormEnabled; else engagement_lead_section">
-              <button class="btn btn-small inverted" (click)="launchContactForm()" type="button">Contact the Project Team</button>
-            </div>
-            <ng-template #engagement_lead_section>
-              <h2>Contacts</h2>
-              <h3>Engagement Lead</h3>
-              <p class="value"><i _ngcontent-c11="" class="material-icons mr-3" role="presentation"
-                  wiid="C937">person</i>{{project?.projectLead?.displayName || "-"}}</p>
-              <p class="value"><i _ngcontent-c11="" class="material-icons mr-3" role="presentation"
-                  wiid="C937">phone</i>{{project?.projectLead?.phoneNumber || "-"}}</p>
-              <p class="value"><i _ngcontent-c11="" class="material-icons mr-3" role="presentation"
-                  wiid="C937">email</i><a class="contacts-link"
-                  href="mailto:{{project?.projectLead?.email}}">{{project?.projectLead?.email || "-"}}</a></p>
-            </ng-template>
-          </div>
-          <div class="inner-col">
-            <button class="btn btn-small inverted" (click)="addEmail()" type="button">Sign up for project updates</button>
+
+        <div class="people-item">
+
+          <button *ngIf="project?.contactFormEnabled" class="btn btn-small contact-anonymous" (click)="launchContactForm()" type="button">
+            <i _ngcontent-c11="" class="material-icons-outlined button-icon" role="presentation" wiid="C937">email</i>
```

Comment:
Often, icons will attempt to be parsed by a screenreader. I'm not sure how material icons handles this, but you may need to add:

```suggestion
            <i _ngcontent-c11="" aria-hidden="true" class="material-icons-outlined button-icon" role="presentation" wiid="C937">email</i>
```

Reply:
I'll read the icons with a screen reader and see what I get

Reply:
I ended up using aria-hidden="true"